content/posts/visual-studio-code-setup.md

```diff
@@ -0,0 +1,142 @@
+
+I tried to stick with one theme for a little while, but I can never make one last. So I keep a handful around and change whenever I feel like it—the `cmd + k cmd + t` shortcut is handy for quick switching.
+
+### Cobalt2
+
+![A Vue.js file with the cobalt2 theme active](./images/theme-cobalt2.png)
+
+A classic blue and yellow [theme](https://marketplace.visualstudio.com/items?itemName=wesbos.theme-cobalt2) from [Wes Bos](https://wesbos.com/). I tend to not use it, as I think it's too bright for me. But it has a great highlight search color when doing a "Find in files".
+
+### Dracula
+
+![A Vue.js file with the Dracula theme active](./images/theme-dracula.png)
+
+Great dark [theme](https://marketplace.visualstudio.com/items?itemName=dracula-theme.theme-dracula) with purples and pinks. This theme puts a solid vertical pink line between editors when you're in split mode, which I really like. So when I am working with two or three splits for long periods of time, I switch to this theme.
+
+### Light owl
+
+![A Vue.js file with the light owl theme active](./images/theme-light-owl.png)
+
+My favorite light theme. Just enough color to be nice, but everything is still readable. It comes with [Night Owl](https://marketplace.visualstudio.com/items?itemName=sdras.night-owl), so a two for one situation.
+
+### Monokai pro
+
+![A Vue.js file with the monokai pro theme active](./images/theme-monokai-pro.png)
+
+This [theme](https://marketplace.visualstudio.com/items?itemName=monokai.theme-monokai-pro-vscode) costs money, but there is a free trial. I'd say it is well worth it since it comes with an icon theme—which I don't use. This is my "home base" theme. I might not use it all the time, but I always come back to it eventually.
+
+
+### Night owl
+
+![A Vue.js file with the night owl theme active](./images/theme-night-owl.png)
+
+A great [theme](https://marketplace.visualstudio.com/items?itemName=sdras.night-owl) for late night coding. [Sarah Drasner](https://sarahdrasnerdesign.com/) really put a lot of TLC into this theme.
+
+### Overnight Slumber Italics
+
+![A Vue.js file with the overnight slumber italics theme active](./images/theme-overnight-slumber-italics.png)
+
+I haven't used this theme very much, but I like the peach and purple vibe of [this theme](https://marketplace.visualstudio.com/items?itemName=cev.overnight). I think I like the "slumber" alternative better than the base theme.
+
+### SynthWave '84
+
+![A Vue.js file with the synth wave '84 theme active](./images/theme-synthwave.png)
+
+SUPER POP COLORS! When I really need to tell the difference between variables and functions, I turn on [this theme](https://marketplace.visualstudio.com/items?itemName=RobbOwen.synthwave-vscode). I don't turn on the custom glow CSS because glow is very distracting to me.
```

Comment:
it would be nice to have a little closing paragraph - thanks for reading, send me your tips/themes, coming next, etc.